app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ApplicationRecord
   require 'digest'
```

Comment:
I commented on this line and its worked without any error.

Reply:
Yeah, we didn't notice that digest already implements in the `has_secure_password` section.